client/src/components/main/Timeline.jsx

```diff
@@ -0,0 +1,101 @@
+import React from 'react';
+import {withGoogleMap, GoogleMap, Marker, Polyline} from 'react-google-maps';
+import { connect } from 'react-redux';
+import { bindActionCreators } from 'redux';
+import { Link } from 'react-router-dom';
+import Markers from './maps/Markers.jsx';
+import Search from './maps/Search.jsx';
+import PostList from './PostList.jsx';
+
+import { 
+  handlePlacesChanged,
+  handleBoundsChanged,
+  handleSearchArea,
+} from '../../store/modules/map';
+
+//The strategy utilized here only works with posts coming from the server organized by timestamp,
+//it cannot use the same routes as the home page.
+
+
+const TimelineComponent = withGoogleMap(props => {
+  const path = [];
+  props.markers.forEach(function (post) {
+    var obj = {'lat': Number(post.lat), 'lng': Number(post.lng)};

```

Comment:
Maybe consider standardizing our way of treating these lat/long strings, either Number() or parseFloat() but not both...